colcon-build/README.md

```diff
@@ -0,0 +1,42 @@
+# colcon-build
+
+This action runs `colcon build`.
+
+## Usage
+
+```yaml
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: ros:galactic
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get modified packages
+        id: get-modified-packages
+        uses: autowarefoundation/autoware-github-actions/get-modified-packages@tier4/proposal
+
+      - name: Build
+        if: ${{ steps.get-modified-packages.outputs.modified-packages != '' }}
+        uses: autowarefoundation/autoware-github-actions/colcon-build@tier4/proposal
+        with:
+          rosdistro: galactic
+          target-packages: ${{ steps.get-modified-packages.outputs.modified-packages }}
+          build-depends-repos: build_depends.repos
+```
+
+## Inputs
+
+| Name                | Required | Description                                     |
+| ------------------- | -------- | ----------------------------------------------- |
+| rosdistro           | true     | ROS distro.                                     |
+| target-packages     | true     | The target packages to build.                   |
+| build-depends-repos | false    | `.repos` file that includes build dependencies. |
```

Comment:
```suggestion
| rosdistro           | true     | The ROS distro.                                     |
| target-packages     | true     | The target packages to build.                   |
| build-depends-repos | false    | The `.repos` file that includes build dependencies. |
```

Reply:
@KeisukeShima Currently, I didn't add `The` to `ROS distro`. If you add it here, please edit other READMEs as well.
But I think it should be another PR.

Reply:
OK. I created an issue.
https://github.com/autowarefoundation/autoware-github-actions/issues/58

Reply:
Thank you. And could you remove it from this PR to make the style consistent?

Reply:
[ad1cfcd](https://github.com/autowarefoundation/autoware-github-actions/pull/49/commits/ad1cfcd952e8065fac795c07c10c11bf9ea1aa40)